packages/ListBox/stories/ListBox.keyboard.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Story } from "storybook/assets/styles/common.styles";
+import { getStoryName } from "storybook/storyTree";
+import Toast from "../../Toast/src";
+
+import {
+  SimpleStory,
+  WithDisabled,
+  WithDividers,
+  WithFilterStory,
+  WithInline,
+  WithRawItems,
+} from "./examples/Keyboard";
+
+const storyName = getStoryName("ListBox");
+
+export default {
+  title: `${storyName}/Backyard/Keyboard`,
```

Comment:
```suggestion
  title: `${storyName}/Backyard/Tests`,
```

Reply:
Why would this be better?

Reply:
It seems to fit better.  The instructions are great – they really facilitate a manual test.  But it's still a testing page.

Reply:
I settled for keyboarTesting just naming test something is hard to find on the search menu of storybook.